addons/event_sale/models/event.py

```diff
@@ -21,6 +21,7 @@ class event_event(models.Model):
     badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
+    is_unlimited = fields.Boolean('Unlimited Tickets', compute='_compute_seats_availability')
```

Comment:
hello  @tde-banana-odoo , currently the calculation of seat_max of event is based on tickets seat_max(total of all tickets seat_max if tickets_type exist(event_Sale installed) ) so you can not define seat_max on event you must specify the seat_max of tickets types. ((i.e issue in yout example if out of 50 , 5 seats available now and then on each and every silver, golf, basic how many seats(quantity) are displayed on event tickets type on website for sale???  )  so i think  the intended behaviour  thanks